src/App.tsx

```diff
@@ -235,9 +227,45 @@ const App: React.FC = () => {
           </Menu.Menu>
         </Menu>
       </header>
-      <Container className={styles.container}>
-        <Tab className={styles.tabs} panes={panes} />
-      </Container>
+
+      <Sidebar.Pushable>
+        <Sidebar
+          as={Menu}
+          visible={visible}
+          vertical
+          inverted={appState.darkMode}
+          animation="overlay"
+          onHide={() => setVisible(false)}
+          width="thin"
+          aria-label="options"
+        >
+          <Settings />
```

Comment:
For <Settings /> <Contact  /> <About /> and <Changelog /> change the trigger to a <Menu.Item /> instead of the <Dropdown.Item />